waterloop-site/src/components/SustainableTech/Teams/TeamsCarousel.tsx

```diff
@@ -0,0 +1,175 @@
+import React from 'react';
+import { useSwipeable } from 'react-swipeable';
+import styled from 'styled-components';
+import useTeamImages from './hooks/team-images';
+
+const TeamsContentWrapper = styled.div`
```

Comment:
I think you forgot to add the drop-shadow in the carousel container, as shown in the Figma design